docker/Dockerfile

```diff
@@ -6,7 +6,6 @@ LABEL description="This dockerfile provides a working environment for \
                    Allen Institute for Brain Science optical physiology data \
                    processing pipelines."
 
-ARG SUITE2P_TAG=v0.10.2
```

Comment:
Just to double check but suite2p version pinning is now being done by the `requirements.txt` in the root of this repository is that right?

Reply:
That is correct